pandas/_libs/hashtable_class_helper.pxi.in

```diff
@@ -192,7 +192,7 @@ cdef class StringVector:
 
         append_data_string(self.data, x)
 
-    cdef extend(self, ndarray[:] x):
+    cdef extend(self, ndarray[object] x):
```

Comment:
im not sure this is accurate, might be a string dtype?  IIRC this class isnt used